### DISCUSSION TITLE
Remove padding from content in grid component

### DISCUSSION THREAD

**Discussion by LouisRoth** [carbon-design-system/carbon-components-svelte#1708] [Q&A] [answered]
I would like to remove the padding, however I am not sure how to do it. I tried using the $$resProps, although I didn't get it to work. In the forum I saw, that you can override classes for the hole codebase, however I want to apply it only to one specific element. 
Is this even possible?

**Comment by metonym** [comment, marked as answer]
There is a `noGutter` prop that removes the padding.

```svelte
<Grid>
  <Row>
    <Column noGutter>...</Column>
  </Row>
</Grid>
```
